content/en/apps/guides/performance/telemetry.md

```diff
@@ -97,4 +97,22 @@ When the aggregate doc is created the Telemetry service also includes a snapshot
 | `dbInfo.idb_attachment_format` | The format of database attachments. |
 | `dbInfo.db_name` | The name of the local database. |
 | `dbInfo.auto_compaction` | Whether or not auto compaction is set. |
-| `dbInfo.adapter` | The database adapter being used. |
\ No newline at end of file
+| `dbInfo.adapter` | The database adapter being used. |
+
+## Export data to JSON
+
+{{% alert title="Note" %}}
+Telemetry data can be viewed directly in your browser with [Fauxton](https://couchdb.apache.org/fauxton-visual-guide/) at `https://{{CHT_INSTANCE_URL}}/_utils`, and navigating the `medic-users-meta` database.
+{{% /alert %}}
+
+To export all telemetry in JSON for further analysis or visualization, first meet these three prerequisites:
+
+1. Open a command line terminal in the directory where you want to save the export files
+1. Ensure that both [`node`](https://nodejs.org/en/) and [`npm`](https://www.npmjs.com/get-npm) are installed and that the needed `node` libraries are installed: `npm install inquirer pouchdb-core fs path minimist pouchdb-adapter-http`
+1. Get a current copy of the export script: `curl -s -o get_users_meta_docs.js https://raw.githubusercontent.com/medic/cht-core/master/scripts/get_users_meta_docs.js` 
+
+To export the data run this command, replacing `USERNAME`, `PASSWORD`, `COUCHDB_SERVER` and `COUCHDB_PORT`  with your own values: `node get_users_meta_docs.js --mode batch --type telemetry https://USERNAME:PASSWORD@COUCHDB_SERVER:COUCHDB_PORT/medic-users-meta > telemetry.json`.  
```

Comment:
Opening the terminal doesn't need to be stated as a prerequisite. It is fine to include as a step if running the export is an explicit step 4, but as it is now I would just remove this step all together, and add a note to the export line. Given that there is an example, I don't think you need to explicitly state all the vars either. 

Also, I don't think we have anything in the doc style guide, but it is nice to have commands on a separate line to make it easier to read and copy & paste.

```suggestion
1. Ensure that both [`node`](https://nodejs.org/en/) and [`npm`](https://www.npmjs.com/get-npm) are installed and that the needed `node` libraries are installed: `npm install inquirer pouchdb-core fs path minimist pouchdb-adapter-http`
1. Get a current copy of the export script: `curl -s -o get_users_meta_docs.js https://raw.githubusercontent.com/medic/cht-core/master/scripts/get_users_meta_docs.js` 

To export the data open a terminal in the folder where you want to save the export, and run this command:
 `node get_users_meta_docs.js --mode batch --type telemetry https://USERNAME:PASSWORD@COUCHDB_SERVER:COUCHDB_PORT/medic-users-meta > telemetry.json` 
```